onnx/test/shape_inference_test.py

```diff
@@ -1970,6 +1970,29 @@ def test_linearclassifier_2D_input(self):  # type: () -> None
                                           make_tensor_value_info('z', TensorProto.FLOAT, (4, 3))],
                                           opset_imports=[make_opsetid('ai.onnx.ml', 1), make_opsetid('', 11)])
 
+    def test_range(self):  # type: () -> None
+        graph = self._make_graph(
+            [('start', TensorProto.FLOAT, ()),
```

Comment:
Need another test for `int` type.

Reply:
This test is for shape inference. I assume you wanted another test for the actual op in `range.py` ? Anyway, I made one of the 2 tests in shape inference pertaining to INT as well.

Reply:
Added a test in 'range.py' pertaining to a node test for int32 and one for shape inference for int32. Resoving this.